walkthroughs/week-2-server/examples/server-stats/src/main/webapp/script.js

```diff
@@ -19,7 +19,7 @@ function getServerStats() {
   fetch('/server-stats').then(response => response.json()).then((stats) => {
     // stats is an object, not a string, so we have to
     // reference its fields to create HTML content
-
+cd
```

Comment:
I would remove this (I'm guessing this was a terminal command, haha).

Reply:
Yes that was definitely a mistake!